hub/api/dataset.py

```diff
@@ -348,7 +352,7 @@ def ingest(
         if len(os.listdir(src)) < 1:
             raise AutoCompressionError(src)
 
-        if images_compression == "auto":
+        if images_compression is "auto":
```

Comment:
Actually == is better for such comparisons

Reply:
yes I agree, was trying to keep this change limited to the auto-compression PR as rightly [suggested](https://github.com/activeloopai/Hub/pull/1117#discussion_r688664083) by @mccrearyd .
It's a tiny change for now so I guess we can make it.